Index: jquery/Makefile
===================================================================
--- jquery.orig/Makefile	2012-05-22 11:55:47.000000000 -0300
+++ jquery/Makefile	2013-08-01 21:44:10.674826432 -0300
@@ -5,8 +5,8 @@
 PREFIX = .
 DIST_DIR = ${PREFIX}/dist
 
-JS_ENGINE ?= `which node nodejs 2>/dev/null`
-COMPILER = ${JS_ENGINE} ${BUILD_DIR}/uglify.js --unsafe
+JS_ENGINE ?= `which node 2>/dev/null || which nodejs 2>/dev/null`
+COMPILER = `which uglifyjs 2>/dev/null` --unsafe
 POST_COMPILER = ${JS_ENGINE} ${BUILD_DIR}/post-compile.js
 
 BASE_FILES = ${SRC_DIR}/core.js\
@@ -42,11 +42,11 @@
 JQ_VER = $(shell cat version.txt)
 VER = sed "s/@VERSION/${JQ_VER}/"
 
-DATE=$(shell git log -1 --pretty=format:%ad)
+DATE=$(shell date)
 
 all: update_submodules core
 
-core: jquery min hint size
+core: jquery min size
 	@@echo "jQuery build complete."
 
 ${DIST_DIR}:
